sensor_msgs/PointCloud2 keypoint_point_cloud
std_msgs/Float64MultiArray descriptor